deploy/playbooks/roles/common/tasks/letsencrypt.yml

```diff
@@ -1,45 +1,44 @@
 ---
+# letsencrypt doesn't recommend using the Ubuntu-provided letsencrypt package
+# https://github.com/certbot/certbot/issues/3538
+# They do recommend using certbot from their PPA for Xenial
+# https://certbot.eff.org/#ubuntuxenial-nginx
+
+- name: install software-properties-common
+  apt:
+    name: software-properties-common
+    state: latest
+    update_cache: yes
+  become: true
+
+- name: add certbot PPA
+  apt_repository:
+    repo: "ppa:certbot/certbot"
+  become: true
+
+- name: install certbot
+  apt:
+    name: python-certbot-nginx
+    state: latest
+    update_cache: yes
+  become: true
 
 - name: ensure letsencrypt acme-challenge path
   file:
     path: "{{ ssl_webroot_path }}"
     state: "directory"
     mode: 0755
-  sudo: yes
-
-- name: unlink app nginx config
-  file:
-    path: "/etc/nginx/sites-enabled/{{ app_name }}.conf"
-    state: "absent"
-  sudo: true
-
-- name: create temporary nginx config
-  template:
-    src: "../templates/nginx_tmp_site.conf"
-    dest: "/etc/nginx/sites-enabled/{{ app_name }}.conf"
-  sudo: true
-
-- name: restart nginx
-  sudo: yes
-  service:
-    name: nginx
-    state: restarted
```

Comment:
It's no longer needed because the .well-known dir is in the live nginx config as of https://github.com/ceph/chacra/commit/fc7f58c4350e408682738d4efb70b9dcaf840075#diff-d4a4bd20d7a2257d097931f40d6545b5

The whole copying the temp config, restarting, then moving the original config back is unnecessary because the nginx code in the temp config is already in the main chacra nginx config.